tests_e2e/pipeline/pipeline-cleanup.yml

```diff
@@ -11,7 +11,9 @@ schedules:
       - develop
     always: true
 
-# no PR triggers
+trigger:
```

Comment:
The cleanup pipeline is being run on PRs. Adding the trigger to make it match the settings of the main pipeline.